packages/spritesheet/test/resources/src/sheetMultipack.tps

```diff
@@ -0,0 +1,249 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm not sure we should add the tps file. I don't think we're adding source files like this in other tests. Thoughts?

Reply:
In that folder, there is already a tps file and the individual sprites. I just followed suit.

Reply:
Sounds good.